README.md

```diff
@@ -55,7 +49,7 @@ Use Airflow to author workflows as directed acyclic graphs (DAGs) of tasks. The
 - [Airflow merchandise](#airflow-merchandise)
 - [Links](#links)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- END doctoc generated TOC please keep comment here to allow auto-update -->
```

Comment:
The inclusion of a dash (`-`) here is causing a test to fail, as this text is auto-generated.